wgsl/index.bs

```diff
@@ -3165,6 +3469,25 @@ where compatibility is defined by the following table.
       <td>[[WebGPU#dom-gpubindingtype-writeonly-storage-texture|writeonly-storage-texture]]
 </table>
 
+If |B| is a [=uniform buffer=] variable in a resource interface,
+and |WB| is the WebGPU GPUBuffer bound to |B|, then:
```

Comment:
Can we link `WebGPU GPUBuffer` back to the spec?

Reply:
Fixed in 9603cf